src/urg_stamped.cpp

```diff
@@ -397,12 +402,24 @@ class UrgStampedNode
     if (status != "00")
     {
       ROS_ERROR("%s errored with %s", echo_back.c_str(), status.c_str());
-      errorCountIncrement();
+      errorCountIncrement(status);
       return;
     }
 
     ROS_DEBUG("Scan data stopped");
   }
+  void cbRB(
+      const boost::posix_time::ptime& time_read,
+      const std::string& echo_back,
+      const std::string& status)
+  {
+    if (status != "00" && status != "01")
+    {
+      ROS_ERROR("%s errored with %s", echo_back.c_str(), status.c_str());
+      ROS_ERROR("Failed to reboot. Please power-off the sensor. Exiting.");
+      ros::shutdown();
```

Comment:
Need to add a small sleep before shutdown, otherwise, `rosout` can't receive the error log.

Reply:
this line is removed since duplicated with the line https://github.com/seqsense/urg_stamped/pull/71/files#diff-570f0f63aadeb6832d4294e9e7c9ce3fR471